include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -977,24 +977,22 @@ struct CustomOpApi {
 
   void ThrowOnError(OrtStatus* result);
 
-  void CreateOpAttr(_In_ const char* name,
-                    _In_ const void* data,
-                    _In_ int len,
-                    _In_ OrtOpAttrType type,
-                    _Outptr_ OrtOpAttr** op_attr);
+  OrtOpAttr* CreateOpAttr(_In_ const char* name,
```

Comment:
>OrtOpAttr* [](http://example.com/codeflow?start=2&length=10)

Should we keep the old version ?

Reply:
Release 1.12 is the first release for the API listed here, it should be fine to further improve it before its inception.